src/mjolnir/ingest_transit.cc

```diff
@@ -685,7 +698,7 @@ void stitch_tiles(const boost::property_tree::ptree& pt,
     do {
 
       // open tile make a hash of missing stop to invalid graphid
-      auto tile = read_pbf(current_path, lock);
+      auto tile = read_pbf(current_path);
```

Comment:
check again..

Reply:
no, this one is fine to not lock @kevinkreiser , just couldn't think of it in the rush of the review. 

this is the tile this thread will process and no other thread will ever write to (only maybe read); the current thread will write this tile only. there's another `read_pbf` further down, which needs locking (where it's trying to find missing nodes in other tiles) and that one is locked.